README.md

```diff
@@ -630,7 +630,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Data USA](https://datausa.io/about/api/) | US Public Data | No | Yes | Unknown |
 | [Data.gov](https://api.data.gov/) | US Government Data | `apiKey` | Yes | Unknown |
 | [Data.parliament.uk](https://explore.data.parliament.uk/?learnmore=Members) | Contains live datasets including information about petitions, bills, MP votes, attendance and more | No | No | Unknown |
-| [Deutscher Bundestag DIP](| [District of Columbia Open Data] (https://dip.bundestag.de/documents/informationsblatt_zur_dip_api_v01.pdf)| This API provides read access to DIP entities (operations and operation items, activities, persons, as well as printed materials and plenary minutes) from the german bundestag | apiKey | Yes | Unknown |
+| [Deutscher Bundestag DIP](https://dip.bundestag.de/documents/informationsblatt_zur_dip_api_v01.pdf)| This API provides read access to DIP entities (operations and operation items, activities, persons, as well as printed materials and plenary minutes) from the german bundestag | apiKey | Yes | Unknown |
```

Comment:
The link is targeted towards a pdf, please make sure you have a webpage regarding the documentation for the API and also limit your description to 100 characters which is the acceptable limit otherwise it will fail the checks and after committing the suggestion please squash your commits into a single one.

Reply:
I have made the description a shorter and squashed my commits. The documentation only does exist as a pdf.